samples/rssreader/macos/Pods/DoubleConversion/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright 2006-2011, the V8 project authors. All rights reserved.
```

Comment:
The `Pods/` folder should _not_ be checked in. It is generated by CocoaPods.

Reply:
Sorry, I didn't realize the default .gitignore doesn't include the Pods folder for MacOS, only for iOS. It should be fixed now.